content-src/asrouter/components/SnippetBase/SnippetBase.jsx

```diff
@@ -7,6 +7,10 @@ export class SnippetBase extends React.PureComponent {
   }
 
   onBlockClicked() {
+    if (this.props.provider === "preview") {
```

Comment:
We don't want to prevent the dismiss action otherwise it might look like a bug to QA.